app/src/main/res/layout/fragment_roadbook.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/fragment_roadbook_directors_parent"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/list"
+        android:name="com.github.factotum_sdp.navdraweractivity.RoadBookFragment"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        app:layoutManager="LinearLayoutManager"
+        tools:context=".ui.roadbook.RoadBookFragment"
+        tools:listitem="@layout/fragment_destrecord">
+
+    </androidx.recyclerview.widget.RecyclerView>
+
+
+    <LinearLayout
+        android:layout_width="64dp"
+        android:layout_height="131dp"
+        android:layout_marginEnd="16dp"
+        android:layout_marginBottom="16dp"
+        android:orientation="vertical"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent">
+
+
+        <com.google.android.material.floatingactionbutton.FloatingActionButton
```

Comment:
The placement of the delete button is awkward

Reply:
As explained in the comment of my PR, that button is purely optional and for a demo purpose. It wasn't a requirement for that task. The delete action will be later triggered by a more sophisticated (more clicks) action of the user, to avoid some wrong delete action.